src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,5 @@
+databaseChangeLog:
```

Comment:
I suggest use the following structure and naming for changelogs (XML or yaml up to you):
```
- app_migrations.xml - contains all changelogs
-- changelog-1.0.0.xml - contains changesets of app with version 1.0.0
-- changelog-1.1.0.xml - contains changesets of app with version 1.1.0
```